packages/accounts-express/accounts_express_client_tests.js

```
@@ -0,0 +1,302 @@
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
```

ESLint

[error] `meteor/accounts-base` import should occur before import of `meteor/meteor`

(import/order)